set.mm

```diff
@@ -50552,9 +50552,12 @@ We have not yet defined relations ( ~ df-rel ), but here we introduce a few
     $d x y z R $.  $d x y z A $.
     $( Define the well-founded relation predicate.  Definition 6.24(1) of
        [TakeutiZaring] p. 30.  For alternate definitions, see ~ dffr2 and
-       ~ dffr3 .  (Contributed by NM, 3-Apr-1994.) $)
+       ~ dffr3 .  A class is called well-founded when the membership relation
+       ( ~ df-eprel ) is well-founded on it (some sources request that the
+       membership relation be well-founded on its transitive closure).
```

Comment:
Maybe "A class is called well-founded when the membership relation ` _E ` (see ~ df-eprel ) is well-founded on it, i.e., ` _E Fr A ` holds for a well-founded class `A` (some sources request that the membership relation be well-founded on its transitive closure)."

Reply:
Done.